pdf/pdfcanvas.h

```diff
@@ -56,6 +57,15 @@ class PDFCanvas : public QQuickItem
     void setSpacing(float newValue);
 
     /**
+     * Getter for property #linkWiggle.
+     */
+    float linkWiggle() const;
+    /**
+     * Getter for property #linkWiggle.
+     */
```

Comment:
Similar already here elsewhere, so ok to have, but can also omit comments as noise.